staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -604,6 +616,64 @@ func TestPatchResourceWithRacingVersionConflict(t *testing.T) {
 	tc.Run(t)
 }
 
+func TestPatchResourceWithInvalidJSONPatch(t *testing.T) {
+	namespace := "bar"
+	name := "foo"
+	uid := types.UID("uid")
+
+	tc := &patchTestCase{
+		name: "TestPatchResourceWithInvalidJSONPatch",
+
+		startingPod: &example.Pod{},
+		updatePod:   &example.Pod{},
+
+		expectedError:     `invalid character 'i' looking for beginning of value`,
+		expectedErrorType: metav1.StatusReasonBadRequest,
+		expectedTries:     1,
+	}
+
+	tc.startingPod.Name = name
+	tc.startingPod.Namespace = namespace
+	tc.startingPod.UID = uid
+	tc.startingPod.ResourceVersion = "2"
+	tc.startingPod.APIVersion = examplev1.SchemeGroupVersion.String()
+	tc.updatePod = tc.startingPod
```

Comment:
Maybe you don't need updatePod at all? It's weird to have `updatePod:   &example.Pod{}` and then `tc.updatePod = tc.startingPod`.

Reply:
Actually why not using `setTcPod` like other test cases?

Reply:
Done.